generators/app/templates/src/containers/Profile/profile.style.js

```diff
@@ -73,6 +73,11 @@ export const FormRenderContainer = styled.div`
     padding: 20px;
     width: 100%;
 
+    div > div > div {
```

Comment:
Could you add a class to the div to apply this style as this looks a bit brittle.

Reply:
Unfortunately not - the div we're altering the style for is part of the components library (a generated form model) so there's no more convenient way to "reach in" to the div we need than this.